incubator/command-dump/src/test/resources/io/aklivity/zilla/runtime/command/dump/internal/FileSystemApplicationIT_shouldReadFileExtensionOnly.txt

```diff
@@ -48,7 +48,7 @@ Zilla Frame
         Type: 
             Length: -1
             Type: 
-        Payload Size: 0
+        Payload Size: -1
```

Comment:
Although payload size defaults to `-1`, this extension-only maps to `head` request, right?
So I think it is legitimate to be explicit about `0` payload size for this.
In general, we are using `-1` to mean unbounded length, and overriding that when the length is known.
In this case we know the length is `0`.